sdk/storage/README.md

```diff
@@ -139,139 +157,181 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
-## SDK Architecture
+## Architecture
 
-The Azure Storage SDK for JavaScript provides low-level and high-level APIs. Take Blob SDK as example:
+The Azure Storage client libraries for JavaScript provides low-level and high-level APIs. Take Blob client library as example:
 
-- ServiceURL, ContainerURL and BlobURL objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
+- ServiceClient, ContainerClient and BlobClient objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
 
 - The high-level APIs provide convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).
 
+## Examples
+
 ## Code Samples
 
 ```javascript
 const {
   BlobServiceClient,
   newPipeline,
-  SharedKeyCredential,
-  AnonymousCredential,
-  TokenCredential
+  SharedKeyCredential
 } = require("@azure/storage-blob");
 
 async function main() {
   // Enter your storage account name and shared key
-  const account = "account";
-  const accountKey = "accountkey";
+  const account = "";
+  const accountKey = "";
 
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use TokenCredential with OAuth token
-  const tokenCredential = new TokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
-
-  // Use AnonymousCredential when url already includes a SAS signature
-  const anonymousCredential = new AnonymousCredential();
-
   // Use sharedKeyCredential, tokenCredential or anonymousCredential to create a pipeline
   const pipeline = newPipeline(sharedKeyCredential);
```

Comment:
>const [](start = 2, length = 5)

Can we use a "non-pipeline" practice here? Create blobServiceClient directly without creating a pipeline

Reply:
Harsha is doing this in #4000.

---
In reply to: [297671270](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r297671270) [](ancestors = 297671270)